### PR TITLE
added hints on limitations around ES with small memory

### PR DESCRIPTION
**Pull request by berndruecker**
Changes required after discussion with @romansmirnov around latest performance observations

### CODE REVIEW

**Comment by berndruecker** [comment]
@romansmirnov WDYT - would that change cut it? 
I work on the sales docs in parallel and make sure they are in-sync...

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 1]
```diff
@@ -126,23 +130,23 @@ Now you can select a hardware package that can cover these requirements. In this
...
 
-However, the following example shows the current configuration of a cluster of size S in Camunda Platform 8 SaaS, which can serve as a starting point for your own sizing. As you can see in the table above, such a cluster can serve 500,000 process instances / day and store up to 5.4 million process instances (in-flight and history).
+s you can see in the table above, such a cluster can serve 500,000 process instances / day and store up to 100 thousand process instances (in-flight and history). You might wonder, why the total number of process instances stored is that low. This is related to limited resources provided to ElasticSearch, yielding in performance problems with too many data stored there. By increasing the available memory to ElasticSearch you can also increase that number. At the same time, even with this rather low number, you can always gurantee the throughput of the core workflow engine during peak loads, as this perfomance is not influenced. Also, you can always increase memory for ElasticSearch later on when if it is required.
```
Comment (romansmirnov) [inline docs/components/best-practices/architecture/sizing-your-environment.md:149]:
```suggestion
As you can see in the table above, such a cluster can serve 500,000 process instances / day and store up to 100 thousand process instances (in-flight and history). You might wonder, why the total number of process instances stored is that low. This is related to limited resources provided to ElasticSearch, yielding in performance problems with too many data stored there. By increasing the available memory to ElasticSearch you can also increase that number. At the same time, even with this rather low number, you can always gurantee the throughput of the core workflow engine during peak loads, as this perfomance is not influenced. Also, you can always increase memory for ElasticSearch later on when if it is required.
```

**Review by christinaausley — CHANGES_REQUESTED** [review]
A few commits with some style guide fixes and a few comments for clarification. Once resolved feel free to re-request review and I'll approve and merge.

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 2]
```diff
@@ -126,23 +130,25 @@ Now you can select a hardware package that can cover these requirements. In this
...
 
+You might wonder why the total number of process instances stored is that low. This is related to limited resources provided to Elasticsearch, yielding performance problems with too much data stored there. By increasing the available memory to Elasticsearch you can also increase that number. At the same time, even with this rather low number, you can always guarantee the throughput of the core workflow engine during peak loads, as this performance is not influenced. Also, you can always increase memory for Elasticsearch later on if it is required.
+
 ### Camunda Platform 8 self-managed
 
-Provisioning Camunda Platform 8 onto your self-managed Kubernetes cluster might depend on various factors. For example, most customes already have own teams providing Elasticsearch for them as a service.
+Provisioning Camunda Platform 8 onto your self-managed Kubernetes cluster might depend on various factors. For example, most customes already have own teams providing Elasticsearch for them as a service. However, the following example shows the current configuration of a cluster of size S in Camunda Platform 8 SaaS, which can serve as a starting point for your own sizing.
```
[the remark this answers is not in the document]
Comment (berndruecker) [reply docs/components/best-practices/architecture/sizing-your-environment.md:149]: Now - "the following example" is referring to the table that comes next

**Comment by akeller** [comment]
@christinaausley are you still waiting for feedback from @berndruecker or @romansmirnov on this PR?

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 3]
```diff
@@ -76,6 +76,10 @@ Furthermore, data is also sent Operate and Optimize, which store data in Elastic
 
 The data you attach to a process instance (process variables) will influence disk space requirements. For example, it makes a big difference if you only add one or two strings (requiring ~ 1kb of space) to your process instances, or a full JSON document containing 1MB.
 
+:::note
+Elasticsearch needs enough memory available to load a big chunk of this data into memory.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/best-practices/architecture/sizing-your-environment.md]: @berndruecker Double checking on this comment -- then we should be good to merge.

**Review by akeller — APPROVED** [review]

**Pull request merged**